### PR TITLE
React capstone project Disney Tribe

### PR DESCRIPTION
**Pull request by AlaaShaheen2**
- [ ] General requirements

- No linter errrors 
- Used the gitflow 
- Documented my work

- [ ] HTML/CSS & JavaScript requirements

- Followed best practices for HTML&CSS
- Followed best practices for JavaScript

- [ ] Project requirements

- Used API
- Design:   Colors, typography: font face, size and weight, layout: composition and space between elements.
- Interactions: Home page ,   Details page 

- [ ] Testing requirements

- Created unit tests for pure functions in your Redux code.
- Created integration tests for your application using the React Testing Library.

### CODE REVIEW

File: src/Tests/Characters.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <Provider store={store}>
+          <Router>
+            <Character />
+          </Router>
+        </Provider>
+      </React.StrictMode>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```
Comment (HAMMAS-SALEEM) [inline src/Tests/Characters.test.js:21]: Testing

- [ ] Your tests are not passing. Kindly make your application work as expected to pass the tests 😊

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Salaam @AlaaShaheen2 :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Descriptive Readme
✔️ Following Github Flow
✔️ Following linters
:heavy_check_mark: Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AlaaShaheen2** [comment]
Hi @HAMMAS-SALEEM ,

Thank you so much for the kind review and for bringing the test failure to my attention.
I have fixed the issue and pushed the changes for the re-review.

Thank you.

File: src/Tests/Characters.test.js (hunk 1 above)
Comment (AlaaShaheen2) [reply src/Tests/Characters.test.js:21]: Screen Shot 2022-10-22 at 9 26 06

**Review by uwadonat — APPROVED** [review]
## Hi @AlaaShaheen2,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
### Highlights
- Good work presentation ✔️✔️
image2

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by AlaaShaheen2** [comment]
@uwadonat  Thank you :)

**Pull request merged**